.github/workflows/ci-go.yaml

```diff
@@ -5,6 +5,28 @@ on:
       - master
   pull_request:
 jobs:
+  shim:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+          node-version: 20.x
+          cache: 'npm'
+    - name: Install
+      run: |
+        npm i
+        npm i -g @vercel/ncc
+        npm run shim
+    - name: verify no git diff
```

Comment:
In what case will there be any diff when running from inside a github action?

Reply:
this is to make sure you didn't forget to run ncc on your local machine